src/main/java/persistence/sql/mapping/ColumnBinder.java

```diff
@@ -82,6 +71,21 @@ private Column createColumn(final Field field) {
         return column;
     }
 
+    private Column getColumn(final Field field, final String columnName, final int sqlType) {
+        int length = 255;
+        boolean nullable = true;
+        boolean unique = false;
+
+        final jakarta.persistence.Column columnAnnotation = field.getAnnotation(jakarta.persistence.Column.class);
+        if (columnAnnotation != null) {
+            length = columnAnnotation.length();
+            nullable = columnAnnotation.nullable();
+            unique = columnAnnotation.unique();
+        }
+
+        return new Column(columnName, sqlType, new Value(field.getType(), sqlType), length, nullable, unique);
```

Comment:
이미 Column class 내부에 length, nullable, unique 의 기본값이 설정되어 있네요
기본값 활용할 수 있도록 Column 생성자 추가하는건 어떻게 생각하세요?
```suggestion
        final jakarta.persistence.Column columnAnnotation = field.getAnnotation(jakarta.persistence.Column.class);
        if (columnAnnotation != null) {
            return new Column(columnName, sqlType, new Value(field.getType(), sqlType), length, nullable, unique);
        }
        return new Column(columnName, sqlType, new Value(field.getType(), sqlType));
```

Reply:
생성자를 나눠서 필요에 따라 호출하는게 훨씬 좋은 코드 같네요.
피드백 감사합니다!